experiments/generic/disk_loss/disk_loss_ansible_logic.yml

```diff
@@ -72,7 +83,12 @@
         ## POST-CHAOS DISK LIVENESS CHECK
         - name: Verify that the disk is connected to node (post)
           include_tasks: "/utils/cloud/gcp/status_disk.yml"
-
+          when: "cloud_platform == 'GCP'"
+        
+        - name: Verify that the disk is connected to node (post)
+          include_tasks: "/utils/cloud/aws/status_disk.yml"
+          when: "cloud_platform == 'AWS'"
+          
         # Checking disk is attached to node
         - debug: 
             msg: echo "disk attached"   
```

Comment:
why echo ?

Reply:
Thanks, @uditgaurav to point out this thing

Reply:
Done